{
	"schema_version": "1.6.0",
	"id": "USN-6159-1",
	"modified": "2023-06-13T14:58:12.818656Z",
	"published": "2023-06-13T14:58:12.818656Z",
	"related": [
		"CVE-2023-28370",
		"UBUNTU-CVE-2023-28370"
	],
	"summary": "python-tornado vulnerability",
	"details": "It was discovered that Tornado incorrectly handled certain redirect.\nAn remote attacker could possibly use this issue to redirect a user to an\narbitrary web site and conduct a phishing attack by having user access a\nspecially crafted URL.\n",
	"affected": [
		{
			"package": {
				"ecosystem": "Ubuntu:Pro:16.04:LTS",
				"name": "python-tornado",
				"purl": "pkg:deb/ubuntu/python-tornado?arch=src?distro=esm-infra/xenial"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "4.2.1-1ubuntu3.1+esm1"
						}
					]
				}
			],
			"versions": [
				"4.2.1-1ubuntu2",
				"4.2.1-1ubuntu3",
				"4.2.1-1ubuntu3.1"
			],
			"ecosystem_specific": {
				"availability": "Available with Ubuntu Pro (Infra-only): https://ubuntu.com/pro",
				"binaries": [
					{
						"binary_name": "python-tornado",
						"binary_version": "4.2.1-1ubuntu3.1+esm1"
					},
					{
						"binary_name": "python3-tornado",
						"binary_version": "4.2.1-1ubuntu3.1+esm1"
					}
				]
			},
			"database_specific": {
				"source": "https://github.com/canonical/ubuntu-security-notices/blob/main/osv/usn/USN-6159-1.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://ubuntu.com/security/notices/USN-6159-1"
		},
		{
			"type": "REPORT",
			"url": "https://ubuntu.com/security/CVE-2023-28370"
		}
	]
}
